src/DropdownStateMixin.js

```diff
@@ -49,7 +49,8 @@ const DropdownStateMixin = {
   handleDocumentClick(e) {
     // If the click originated from within this component
     // don't do anything.
-    if (isNodeInRoot(e.target, React.findDOMNode(this))) {
+    let target = e.target || e.srcElement;
```

Comment:
Can you add a comment above this line stating that `e.srcElement` is specifically for IE8? This will allow us to deprecate this easier when Bootstrap v4 comes out.